packages/selection/frontend/sirius-components-selection/src/SelectionDialogTreeView.tsx

```diff
@@ -28,16 +28,16 @@ const useTreeStyle = makeStyles()((theme) => ({
   },
 }));
 
+const initialState: SelectionDialogTreeViewState = {
```

Comment:
Why?

Reply:
This can be put outside the component to improve readability.
It does not affect the code here since it's used for the state, seems like a good practice to put values that never change outside the component.